### DISCUSSION TITLE
Chapter 5: Effectful streams

### DISCUSSION THREAD

**Discussion by gvolpe** [gvolpe/trading#16] [General]
## Chapter 5: Effectful streams

⚠️ Note that the order and number of chapters is not yet fully decided, so expect changes here as well ⚠️

**Comment by PaNaVTEC** [comment]
Hi Gabriel! Just a comment, `sharding` is introduced as a concept in 5.4.2.1 but there was no mention or explanation before, or at least I couldn't find it, would you mind making any mention as you did with topic compaction in chapter 3? I think it could help readers understand concepts if you're not familiar with EDA. Thanks for your work!

**Reply by gvolpe** [reply]
Thanks for the feedback, I'll have a look and make sure it's properly introduced to be consistent with the rest of the book.
